### ISSUE TITLE
add coefficient correlation as ES for TOSTMeta

### ISSUE THREAD

**Issue by PrioloDaniel** [Lakens/TOSTER#45]
It would be great if we could run an equivalence test with meta-analysis data using correlation coefficient as an effect size

**Comment by LukasWallrich** [comment]
Yes please - @Lakens any chance that this could be implemented, or that you could provide any guidance on how to do this at present?
